src/relay.cpp

```diff
@@ -12,6 +12,7 @@
 #include "dbconnector.h" 
 #include "configInterface.h"
 
+#define BUFFER_SIZE 9200
```

Comment:
Suggest not use static defined array, as it will use .BSS region which is close to other global data, if memory corruption happened it may easy cause double failure which is hard to track.

Instead we can define a buffer pointer, and use malloc for address allocation which  will use the memory of heap region.  Maybe we can add this into code refine plane.